src/aiortc/rtcrtpsender.py

```diff
@@ -288,21 +327,80 @@ def _send_keyframe(self) -> None:
         Request the next frame to be a keyframe.
         """
         self.__force_keyframe = True
-
+    
+    def ml_predict(self, frame, counter):
+        frame_array = frame.to_ndarray()
+        frame_array = np.resize(frame_array, (1024, 1024, 3))
```

Comment:
why do you need to resize? shouldn't it already be of the right size?

Reply:
`frame.to_ndarray()` is in size of `(1024, 1026, 3)` to my surprise

Reply:
That is really weird, and I would debug as to why this is the case. Resizing it will result in some artifacts if there is some random extra pixels. I'd visualize the frame to see if there's some excess/trace its size.

Reply:
I removed the function for now as it's not being used in the branch.

Reply:
Can you double check that outside of this function this isn't an issue? Verify that we're not getting the wrong size frame for saving/displaying.